.buildkite/pipeline.yml

```diff
@@ -94,30 +94,27 @@ steps:
               context: "UI Tests (iPad)"
 
   #################
-  # Lint Translations
+  # Linters
   #################
-  - label: "🧹 Lint Translations"
-    command: "gplint /workdir/WordPress/Resources/AppStoreStrings.po"
-    plugins:
-      - docker#v3.8.0:
-          image: "public.ecr.aws/automattic/glotpress-validator:1.0.0"
-    agents:
-      queue: "default"
-    notify:
-      - github_commit_status:
-          context: "Lint Translations"
-
-  #################
-  # Lint Ruby Tooling
-  #################
-  #
-  # TODO: This ought to be run on a Docker or Ubuntu agent as there's nothing that requires macOS.
-  # It's currently in the stock setup because... it was faster to build a proof of concept for it this way;
-  # I don't know how to pass the `DANGER_GITHUB_API_TOKEN` secret that we need to a Docker agent.
-  #
-  # This step uses Danger to run RuboCop, but it's "agnostic" about it.
-  # That is, it outwardly only mentions RuboCop, not Danger
-  - label: ":rubocop: Lint Ruby Tooling"
-    command: .buildkite/commands/rubocop-via-danger.sh
-    env: *common_env
-    plugins: *common_plugins
+  - group: " Linters"
```

Comment:
nit: why the extra space? 🙃

Reply:
Ahah. Must be a leftover from me trying to add an emoji or something... 🤔 

Luckily Buildkite trims it in the UI

Reply:
Easy fix... https://github.com/wordpress-mobile/WordPress-iOS/pull/19008